cmake_modules/ccache.cmake

```diff
@@ -16,9 +16,9 @@ if(CCACHE)
     if(APPLE)
         set(CMAKE_CXX_COMPILER "/usr/local/Cellar/ccache/3.7.7/libexec/${CXX_COMPILER}")
     else()
-        set(CMAKE_CXX_COMPILER "/lib/ccache/bin/${CXX_COMPILER}")
+        set(CMAKE_CXX_COMPILER "/lib/ccache/${CXX_COMPILER}")
```

Comment:
Why?

Reply:
This file should not need to be changed

Reply:
Because this was a setting i set when i still used archlinux, ubuntu has it in /lib/ccache/{}

```
 ~  la /lib/ccache
total 16K
drwxr-xr-x   2 root root 4,0K sep  7 09:20 ./
drwxr-xr-x 149 root root  12K sep 11 11:14 ../
lrwxrwxrwx   1 root root   16 jun 15 11:26 c++ -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 c89-gcc -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 c99-gcc -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 cc -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 clang -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 clang++ -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 clang++-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 clang-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 g++ -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 sep  7 09:20 g++-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 g++-9 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 gcc -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 sep  7 09:20 gcc-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 gcc-8 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 gcc-9 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 x86_64-linux-gnu-g++ -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 sep  7 09:20 x86_64-linux-gnu-g++-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 x86_64-linux-gnu-g++-9 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 x86_64-linux-gnu-gcc -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 sep  7 09:20 x86_64-linux-gnu-gcc-10 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 x86_64-linux-gnu-gcc-8 -> ../../bin/ccache*
lrwxrwxrwx   1 root root   16 jun 15 11:26 x86_64-linux-gnu-gcc-9 -> ../../bin/ccache*
```